litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -321,15 +307,23 @@ const DashboardPage: React.FC = () => {
                   history.push('/analytics/dashboard/configure');
                 }}
               >
-                <div>Re-configure dashboard</div>
+                <div>
+                  {t(
+                    'analyticsDashboard.monitoringDashboardPage.reConfigureDashboard'
+                  )}
+                </div>
```

Comment:
May we use Typography or Litmus-ui Text component here

Reply:
But there is no text decoration needed here, is there any benefit of using that?

Reply:
If it is heading then it should be within the header, similarly for the paragraph, and subtitle. This makes it uniform and more adaptive to any future design change.

Reply:
It’s a button text

Reply:
Let’s not make all the changes in bug fix PRs a refactor work will be done as a part of v2 implementation which is scope of beta 2

Reply:
As its a button we may just remove the div and pass it as children of buttonFilled